commons/src/main/java/io/aiven/kafka/connect/common/config/AivenCommonConfig.java

```diff
@@ -53,12 +64,79 @@ public class AivenCommonConfig extends AbstractConfig {
     private static final String GROUP_RETRY_BACKOFF_POLICY = "Retry backoff policy";
     public static final String KAFKA_RETRY_BACKOFF_MS_CONFIG = "kafka.retry.backoff.ms";
 
+    @SuppressWarnings({ "PMD.this-escape", "PMD.ConstructorCallsOverridableMethodcls" })
     protected AivenCommonConfig(final ConfigDef definition, final Map<?, ?> originals) {
         super(definition, originals);
         // TODO: calls getOutputFields, can be overridden in subclasses.
-        validate(); // NOPMD ConstructorCallsOverridableMethod
+        validate(); // NOPMD
     }
 
+    protected static int addFileConfigGroup(final ConfigDef configDef, final String groupFile, final String type,
```

Comment:
I also have a PR open https://github.com/Aiven-Open/cloud-storage-connectors-for-apache-kafka/pull/312 that will split AivenCommonConfig into a Source and Sink Common config so that we can start building the Source connectors to start reading back the data we have stored here, This is mostly a minor structural change but just wanted to give you a heads up. This info does make sense to be moved to the Sink Common config imo.